var/spack/repos/builtin/packages/libpciaccess/package.py

```diff
@@ -36,15 +36,3 @@ class Libpciaccess(Package):
     depends_on('libtool', type='build')
     depends_on('pkg-config@0.9.0:', type='build')
     depends_on('util-macros', type='build')
-
-    def install(self, spec, prefix):
-        # libpciaccess does not support OS X
```

Comment:
ok, i presume this PR https://github.com/LLNL/spack/pull/3457 should keep packages that need MPI working on macOS as `libpciaccess` is directly used only in 
```
$ ag "libpciaccess" var/spack/repos/builtin/packages/
var/spack/repos/builtin/packages/hwloc/package.py
52:    depends_on('libpciaccess')

var/spack/repos/builtin/packages/intel-gpu-tools/package.py
44:    depends_on('libpciaccess@0.10:')

var/spack/repos/builtin/packages/libdrm/package.py
40:    depends_on('libpciaccess@0.10:')

var/spack/repos/builtin/packages/mvapich2/package.py
97:    depends_on('libpciaccess')
```